src/features/duplicates/hooks/useDuplicatesMutations.tsx

```diff
@@ -11,15 +21,21 @@ export default function useDuplicatesMutations(
   const apiClient = useApiClient();
   const dispatch = useAppDispatch();
 
-  type ZetkinDuplicatePatchBody = Partial<
+  type PotentialDuplicatePatchBody = Partial<
     Omit<PotentialDuplicate, 'id' | 'dismissed'>
   > & {
     dismissed?: boolean;
   };
 
+  type PotentialDuplicatePostBody = {
```

Comment:
The name implies that this is the POST body for a "potential duplicate", but I don't think that's an accurate way to describe it. This is being posted to the "merges" API. A better name for this type would be something along the lines of `MergePostBody`.

The two APIs for potential duplicates and merges are very much related, but they are still two separate APIs and should be thought of as such.